pkg/kubelet/kuberuntime/logs/logs_test.go

```diff
@@ -240,7 +241,10 @@ func TestReadRotatedLog(t *testing.T) {
 			Follow: true,
 		}
 		opts := NewLogOptions(&podLogOptions, time.Now())
-		ReadLogs(ctx, file.Name(), containerID, opts, fakeRuntimeService, stdoutBuf, stderrBuf)
+		mu.Lock()
```

Comment:
you don't need a mutex, just capture the variable out of the gorutine is enough

Reply:
Thanks for pointing it out. It is a better approach, I will do the necessary changes.